junebug/router/from_address.py

```diff
@@ -135,6 +136,11 @@ def setup_router(self):
             self.consume_destination(
                 destination['id'], self.handle_outbound_message)
 
+    def get_destination_channel(
+            self, destination_id, message_body=None, message_id=None):
+        config = self.get_static_config()
+        return succeed(str(config.channel))
```

Comment:
Can we add a test for this method? Since we don't use the from address router in our API tests.